docs/_sources/chapters/data-and-variables/operations.rst.txt

```diff
@@ -0,0 +1,213 @@
+Operations
+==========
+
+Operators and Operands
+----------------------
+
+Now that we can store data in variables, let's explore how we can generate new data from existing data.
+
+.. index:: ! operator, ! operand
+
+An **operator** is one or more characters that represents a computation like addition, multiplication, or division. The values an operator works on are called **operands**.
+
+The following are all legal JavaScript expressions whose meaning is more or less clear:
+
+.. sourcecode:: js
+   :linenos:
+
+    let hour = 5;
+    let minute = 120;
+
+    20 + 32;
```

Comment:
It's slightly weird that these expressions aren't being logged or assigned to a variable.